Add IPVLAN and DUMMY to check-config.sh

coolljt0725 opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/docker/docker/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

For additional information on our contributing process, read our contributing
guide https://docs.docker.com/opensource/code/

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- Description for the changelog**

<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->

**\- A picture of a cute animal (not mandatory but encouraged)**

This commit add DUMMY and IPVLAN to check-config.sh
because they are need for ipvlan and macvlan network
driver.
ping @mavenugo 

Signed-off-by: Lei Jitang leijitang@huawei.com

jfrazelle commented:
looks good to me

justincormack commented on contrib/check-config.sh:
```diff
@@ -183,7 +183,7 @@ flags=(
 	CGROUPS CGROUP_CPUACCT CGROUP_DEVICE CGROUP_FREEZER CGROUP_SCHED CPUSETS MEMCG
 	KEYS
-	MACVLAN VETH BRIDGE BRIDGE_NETFILTER
+	VETH BRIDGE BRIDGE_NETFILTER 
```
there is a trailing space here....

jfrazelle replied:
whoa @tianon has found a new partner in crime ;)

On Thu, Jul 7, 2016 at 8:09 AM, Justin Cormack <notifications@github.com>
wrote:

> In contrib/check-config.sh
> <https://github.com/docker/docker/pull/24301#discussion_r69924213>:
>
> > @@ -183,7 +183,7 @@ flags=(
> >  	DEVPTS_MULTIPLE_INSTANCES
> >  	CGROUPS CGROUP_CPUACCT CGROUP_DEVICE CGROUP_FREEZER CGROUP_SCHED CPUSETS MEMCG
> >  	KEYS
> > -	MACVLAN VETH BRIDGE BRIDGE_NETFILTER
> > +	VETH BRIDGE BRIDGE_NETFILTER
>
> there is a trailing space here....
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/24301/files/04a8805413da130382d2f95f2a612fede30ab385#r69924213>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYNbPBQR9U9_sVyKc3_-b0jrLrWESLOks5qTRaggaJpZM4JEFn3>
> .
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

thaJeztah replied:
LOL

tianon replied:
:heart: !

coolljt0725 committed:
Add IPVLAN and DUMMY to check-config.sh

This commit add DUMMY and IPVLAN to check-config.sh
because they are need for ipvlan and macvlan network
driver.

Signed-off-by: Lei Jitang <leijitang@huawei.com>

thaJeztah commented:
LGTM

@justincormack PTAL

vdemeester commented:
LGTM 🐯

justincormack commented:
LGTM
